bigtable/spark/README.md

```diff
@@ -0,0 +1,85 @@
+# Spark Applications for Cloud Bigtable
+
+## Overview
+
+The project shows how to load or save data to [Cloud Bigtable](https://cloud.google.com/bigtable) using [Apache Spark](https://spark.apache.org/) and [Apache HBase™ Spark Connector](https://github.com/apache/hbase-connectors/tree/master/spark) . 
+
+The project uses [sbt](https://www.scala-sbt.org/) as the build tool.
+
```

Comment:
Can you add a paragraph explaining how all of the project interact?

something along the lines of: Spark is the execution environment that can parallelize data processing. The spark provides an api for storage system to plug into this environment. The HBase Spark Connector implements an adapter for Hbase. bigtable-hbase-2.x-hadoop provides a bridge from the HBase api to cloud bigtable. Thus allowing spark pipelines to interact with bigtable using the native spark api.

Reply:
done